packages/block-library/src/section/index.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import {
+	getColorClassName,
+	withColors,
+	InspectorControls,
+	InnerBlocks,
+	PanelColorSettings,
+} from '@wordpress/editor';
+
+export const name = 'core/section';
```

Comment:
I think "container" is a better name, since "section" implies the usage of a `<section>` element, but often you just want to use a simple container just to add a background to something that is *not* semantically a section. Issues to add color options to other blocks keep being rejected because of the assumption that a generic container block could accomplish the same thing. But if the block is limited to using the `<section>` element, then that limits its usage, unless you want to encourage bad semantics.